Makefile

```diff
@@ -206,6 +206,9 @@ endif
 TEST_FLAGS ?= -v
 .PHONY: unit build-unit-tests
 unit unit-test:
+	curl -L https://go.kubebuilder.io/dl/2.3.1/${GOOS}/${GOARCH} | tar -xz -C /tmp/
+	mv /tmp/kubebuilder_2.3.1_${GOOS}_${GOARCH} /usr/local/kubebuilder
+	export PATH=$PATH:/usr/local/kubebuilder/bin
```

Comment:
I think unit tests for vsphereparavirtual requires a local kubebuilder binary
the kubebuilder scaffolding tests do run a local controlplane with etcd
https://gitlab.eng.vmware.com/core-build/guest-cluster-controller/blob/master/pkg/controller/managedcluster/managedcluster_controller_suite_test.go#L41
The test suite for vSphere paravirtual uses “sigs.k8s.io/controller-runtime/pkg/envtest” which stands up a local controlplane with etcd.
So I’m adding this into Makefile to make unit test running

Reply:
I think it would be a worthwhile exercise as a follow-up to refactor all the tests to not use ginkgo/envtest and use standard unit tests + fake client instead. I don't think we  want to carry the controller-runtime dependency here for long because it will conflict with client-go and other kubernetes modules.

Reply:
got it!